.buildkite/bk.fips-integration.pipeline.yml

```diff
@@ -0,0 +1,217 @@
+        depends_on:
+          - packaging-ubuntu-x86-64-fips
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} false
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_X84_64_FIPS}"
+          instanceType: "m5.2xlarge"
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+          # FIPS tests don't have other groups enabled in order to save resources
+
+      - label: "fips:x86_64:sudo:{{matrix}}"
+        depends_on:
+          - packaging-ubuntu-x86-64-fips
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step packaging-ubuntu-x86-64-fips
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_X84_64_FIPS}"
+          instanceType: "m5.2xlarge"
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+
+      - label: "fips:arm64:non-sudo:{{matrix}}"
+        depends_on:
+          - packaging-ubuntu-arm64-fips
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-arm64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} false
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "aws"
+          image: "${IMAGE_UBUNTU_2404_ARM_64}"
+          instanceType: "m6g.2xlarge"
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+
+      - label: "fips:arm64:sudo:{{matrix}}"
+        depends_on:
+          - packaging-ubuntu-arm64-fips
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step packaging-ubuntu-arm64-fips
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "aws"
+          image: "${IMAGE_UBUNTU_2404_ARM_64}"
+          instanceType: "m6g.2xlarge"
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+
+  - group: "fips:Kubernetes"
```

Comment:
fips kubernetes??? are we sure???

Reply:
not sure, added for completeness (let's see if it works, or if all tests will skip)

Reply:
@pkoutsovasilis  I think it will make it easier to distinguish these tests from the regular K8s tests in Agent CI.  We can be more descriptive if the name is confusing.

Reply:
the name is not confusing, the actual testcase is what confuses me a bit. However, since this is deliberate then no objection gents, go on 🙂